src/applications/disability-benefits/all-claims/tests/testUtils.js

```diff
@@ -0,0 +1,37 @@
+import { $ } from '@department-of-veterans-affairs/platform-forms-system/ui';
+
+/**
+ * Input a string value into a va-text-input component.
+ * @param {any} container - React Testing Library container
+ * @param {string} value - String value to enter in the input field
+ * @param {string} selector - select or element
```

Comment:
These two functions look great. Maybe can be moved somewhere more generic so the entire vets-website can use them?

Reply:
Thanks @ataker! These functions are actually just encapsulating logic seen elsewhere in the repo. We need to get this pr merged now, but I'll make sure to move these over in the next ticket already in progress.